File: 3_found-of-objc-app-dev/week-1/Hello World/Hello World/ViewController.m
//
//  ViewController.m
//  Hello World
//
//  Created by Suzy Lee on 7/29/20.
//  Copyright © 2020 Suzy Lee. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UILabel *heightText;
@property (weak, nonatomic) IBOutlet UIButton *buttonText;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    _heightText.text = @"5 foot 3";
    [_buttonText setTitle:@"Convert height to cm!" forState: UIControlStateNormal];
    
}

- (IBAction)pressHelloButton:(UIButton *)sender {
    int heightFeet = 5;
    int heightInches = 3;
    double heightCm = ((heightFeet * 12.0) + heightInches)*2.54;
    
    _heightText.text = [NSString stringWithFormat:@"%fcm", heightCm];
}

@end

File: section-10/Destini-LayoutPractice-iOS13-master/Destini-iOS13/Controller/ViewController.swift
//
//  ViewController.swift
//  Destini-iOS13
//
//  Created by Angela Yu on 08/08/2019.
//  Copyright © 2019 The App Brewery. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    @IBOutlet weak var option1Button: UIButton!
    @IBOutlet weak var option2Button: UIButton!
    @IBOutlet weak var storyText: UILabel!
    
    var storyBrain = StoryBrain()
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        option1Button?.titleLabel!.numberOfLines = 0;
        option2Button?.titleLabel!.numberOfLines = 0;

        updateUI()
    }
    
    
    @IBAction func chooseOption(_ sender: UIButton) {
        let selectedOpt = sender.titleLabel!.text!
        print(selectedOpt)
        
        // Progress to the next story
        storyBrain.nextStory(userChoice: selectedOpt)
        updateUI()
    }
    
    func updateUI() {
        let currStory = storyBrain.getCurrStory()
        
        storyText.text = currStory.title
        option1Button.setTitle(currStory.choice1, for:.normal)
        option2Button.setTitle(currStory.choice2, for:.normal)
    }
    

}

